Yank/repex.py

```diff
@@ -1181,7 +1237,9 @@ class while the simulation is running. This reads the SamplerStates of every
     # Constructors.
     # -------------------------------------------------------------------------
 
-    def __init__(self, mcmc_moves=None, number_of_iterations=1, replica_mixing_scheme='swap-all'):
+    def __init__(self, mcmc_moves=None, number_of_iterations=1, replica_mixing_scheme='swap-all',
+                 online_analysis_interval=None, online_analysis_target_error=1.0,
```

Comment:
That's also a fine option. We just have not really had the ability to easily make a system like that for YANK tests pre-API. Since we have the API in now, its worth putting in. I'll come up with something for tomorrow and see if it works. I really don't want to upload more than a few megs, and maybe it should not even be distributed with the package on conda, kept on a `dev` side of things GitHub or on its own label